qstack/spahm/rho/bond.py

```diff
@@ -39,7 +39,7 @@ def bond(mols, dms,
     elements, mybasis, qqs0, qqs4q, idx, M = dmbb.read_basis_wrapper(mols, bpath, only_m0, printlevel,
                                                                      elements=elements, cutoff=cutoff,
                                                                      pairfile=pairfile, dump_and_exit=dump_and_exit, same_basis=same_basis)
-    if spin is None:
+    if np.array(spin==None, ndmin=1).all():
```

Comment:
It's kind of dirty!
We should probably change it/ make it more clear when we change the arguments!

Reply:
can we have smth like `(spin is None) or all(spin==None)`? not sure it'll work though

Reply:
oudated now !